8dev_restapi.js

```diff
@@ -15,6 +15,24 @@ class Endpoint extends EventEmitter {
     this.transactions = {};
     this.observations = {};
 
+    this.service.on('register', (resp) => {
```

Comment:
refractor resp -> id

Reply:
'id' is already declared in the upper scope 
(Konstruktoriuje)